.github/workflows/cli.yml

```diff
@@ -4,9 +4,10 @@ on:
   workflow_dispatch:
     inputs:
       release_type:
-        required: true
-        type: string
         default: latest
+        required: true
+        type: choice
+        options: [latest, stable, pr]
```

Comment:
Nice! I didn't know that was an option in workflows!

Reply:
only in `on: workflow_dispatch` though I think